app/models.py

```diff
@@ -1820,32 +1836,38 @@ def hypothetical_applications_closed_at(self):
 
     @hybrid_property
     def applications_closed_at(self):
```

Comment:
do we still need these hybrid properties?  everything could just use `closed_at`